src/types.ts

```diff
@@ -68,6 +68,15 @@ export const asDbCurrencyCodeMappings = asObject({
   mappings: asCurrencyCodeMappings
 })
 
+export const asStandardPluginParams = asObject({
+  settings: asObject({
+    latestIsoDate: asOptional(asString, '0')
+  }),
+  apiKeys: asObject({
+    apiKey: asOptional(asString)
+  })
+})
+
```

Comment:
This could have been done in an earlier commit, but at least it's being done now.